release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3808,4 +3808,23 @@ $ oc adm release info 4.6.61 --pullspecs
 [id="ocp-4-6-61-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-62"]
+=== RHSA-2022:7306 - {product-title} 4.6.62 bug fix update
```

Comment:
Interesting, 4.6.62 is showing in the title here, but not rendering in the preview build:

![image](https://user-images.githubusercontent.com/96480979/200056141-dfb1e15a-6f66-4d19-a708-c51da2e39339.png)

Reply:
I pushed an update when I caught the missing 2 and didn't update the preview.